### ISSUE TITLE
Create LICENSE.txt

### ISSUE THREAD

**Issue by jxstanford** [FortAwesome/Font-Awesome#5281]
In order to properly use Font Awesome in a project, the MIT license should be customized to have a copyright statement (for code project).  

It would be appropriate to put that customized file in a LICENSE.txt at the root of your github project.

**Comment by Naatan** [comment]
The font license used by this project says that the license and copyright must remain in-tact, but this is impossible as neither a license file nor a copyright are provided. I'd love to abide by the license but it's currently impossible.

**Comment by 153957** [comment]
It is not really impossible, links to the licences are clearly provided.

**Comment by abelspry** [comment]
Yeah, same issue as @Naatan. The real problem for me is that I'm using npm to download and use font-awesome. I can't just add some text file into the module folder because of we have the links. It's not the right way. npm is part of a deploy process, it's not possible to proceed *adding files by hand* for every deploy. Furthermore, licenses are different for different contents, and that, legally, is a bit painful, because you need to add the licence in the right folder only, avoiding misundestanding:

- `/fonts` folder needs http://scripts.sil.org/OFL
- all that remains use MIT

My company would like to embed font-awesome in the project that sells, but actually it can't because of licenses. The only way to add text licenses is to create a tool that downloads them and put wherever they should placed: no way.

My proposal is to add the MIT licence to the github root folder AND add a http://scripts.sil.org/OFL licence only in the `/font` folder.

**Comment by ilmente** [comment]
Yeah, same issue as @Naatan. The real problem for me is that I'm using npm to download and use font-awesome. I can't just add some text file into the module folder because *I have the links*. It's not the right way. npm is part of a deploy process, it's not possible to proceed *adding files by hand* for every deploy. Furthermore, licenses are different for different contents and that, legally, is a bit painful because you need to add the licence in the right folder only, avoiding misundestanding:

- `/fonts` folder needs http://scripts.sil.org/OFL
- all that remains use MIT

My company would like to embed font-awesome in the project that sells, but actually it can't because of licenses. The only way to add text licenses is to create a tool that downloads them and put wherever they should be placed: no way.

My proposal is to add the MIT licence to the github root folder AND add a http://scripts.sil.org/OFL licence only in the `/font` folder.

**Comment by ilmente** [comment]
I'm proposing a different solution:
- `LICENSE-SUMMARY.txt` contains all the informations about licenses and where they are applied;
- `licenses/` folder contains all the license files with updated copyright holder.

Let me know if this makes sense, otherwise ignore it.
Thanks.

**Comment by ddekany** [comment]
Same problem here... Providing a link to SIL Open Font license it is not enough, as that's just a template. It starts like this:

> Copyright (c) <dates>, <Copyright Holder> (<URL|email>),
> with Reserved Font Name <Reserved Font Name>.

So I believe you should copy-paste the template, fill the blanks, create a TXT from it, and link to that (or, just insert the license on the license Web page).

The Apache policy requires including the licenses of all subcomponents. But I can't find the license anywhere, only its template.

**Comment by tagliala** [comment]
maybe a license file is good for the github badge, but there are several licenses involved here.

I don't know what should be the primary one.

https://help.github.com/articles/adding-a-license-to-a-repository/

**Comment by ddekany** [comment]
There's no such thing as a primary one. Different files (or rather directories in your case) are covered by different licenses. You could, for example, put all the licenses into the same single LICENSE file, also describing there which license applies to which files. (Apache projects used to do that, because it's quite normal that your product includes files with various licenses.)

**Comment by JD-Robbs** [comment]
Agreed, the SIL OFL actually refers to the "copyright notice above" - so for those who, for their purposes, are required to include the license - we'd need an officially filled-out template.

**Comment by sensibleworld** [comment]
Doing a little management in Github and (batch) closing out potentially stale issues. If the above still needs attention, please let us know and we’ll re-open!
